src/bindings/python/src/pyopenvino/graph/ops/constant.cpp

```diff
@@ -72,20 +26,21 @@ py::buffer_info _get_buffer_info(const ov::op::v0::Constant& c) {
                            py::format_descriptor<T>::format(),               /* Python struct-style format descriptor */
                            static_cast<size_t>(shape.size()),                /* Number of dimensions */
                            std::vector<size_t>{shape.begin(), shape.end()},  /* Buffer dimensions */
-                           _get_byte_strides<T>(shape)                       /* Strides (in bytes) for each index */
+                           Common::constant_helpers::_get_byte_strides<T>(shape) /* Strides (in bytes) for each index */
     );
 }
 
 // TODO: Remove in future and re-use `get_data`
 template <>
 py::buffer_info _get_buffer_info<ov::float16>(const ov::op::v0::Constant& c) {
     ov::Shape shape = c.get_shape();
-    return py::buffer_info(const_cast<void*>(c.get_data_ptr()),              /* Pointer to buffer */
-                           static_cast<size_t>(c.get_element_type().size()), /* Size of one scalar */
-                           std::string(1, 'H'),                              /* Python struct-style format descriptor */
-                           static_cast<size_t>(shape.size()),                /* Number of dimensions */
-                           std::vector<size_t>{shape.begin(), shape.end()},  /* Buffer dimensions */
-                           _get_byte_strides<ov::float16>(shape)             /* Strides (in bytes) for each index */
+    return py::buffer_info(
+        const_cast<void*>(c.get_data_ptr()),                            /* Pointer to buffer */
+        static_cast<size_t>(c.get_element_type().size()),               /* Size of one scalar */
+        std::string(1, 'H'),                                            /* Python struct-style format descriptor */
+        static_cast<size_t>(shape.size()),                              /* Number of dimensions */
+        std::vector<size_t>{shape.begin(), shape.end()},                /* Buffer dimensions */
```

Comment:
Why make copy? the ov::Shape can be casted to this kind of vector is base class for Shape.